packages/core/src/docdb/commands/addRegion.ts

```diff
@@ -0,0 +1,96 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import { getLogger } from '../../shared/logger'
+import { telemetry } from '../../shared/telemetry'
+import { localize } from '../../shared/utilities/vsCodeUtils'
+import { DBClusterNode } from '../explorer/dbClusterNode'
+import { DefaultDocumentDBClient } from '../../shared/clients/docdbClient'
+import { ToolkitError } from '../../shared'
+import { showViewLogsMessage } from '../../shared/utilities/messages'
+import { isValidResponse } from '../../shared/wizards/wizard'
+import { CancellationError } from '../../shared/utilities/timeoutUtils'
+import { CreateGlobalClusterWizard } from '../wizards/createGlobalClusterWizard'
+import { CreateDBClusterMessage } from '@aws-sdk/client-docdb'
+import { createInstancesForCluster } from './createCluster'
+
+export async function addRegion(node: DBClusterNode): Promise<void> {
+    if (!node) {
```

Comment:
What is this checking for? The `node: DBClusterNode` parameter does not seem to be nullable. Can the value be falsy?

Reply:
Commands can be invoked indirectly anywhere in vscode, so this is just a safety check.